### PR TITLE
Organization & Loops Practice

### PR DESCRIPTION
**Pull request by bfwetmore**
- Add Directories for each lesson.
- Add Sub directories for challenges and practices
- Modify app.js file to a for loop

### CODE REVIEW

File: Loops/Loops Practice/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="utf-8">
+    <title>Practice JavaScript Loops</title>
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+<main></main>
+<script src="js/app.js"></script>
+</body>
+</html>
```
Comment (do-work) [inline Loops/Loops Practice/index.html]: I would start adding new lines at the end of file.

**Review by do-work — CHANGES_REQUESTED** [review]
Keep in mind common naming conventions and note that each language can have different conventions.
A lot of times I will look at a specific language 'topic' on Github - https://github.com/topics/javascript
From there I will browse for some of the top projects and look at their structure.
One example is https://github.com/axios/axios
Also you can look for language specific 'style guides'. 
This may have more info than what your looking for right now but the takeaway to me is that the file and dir names should be lowercase.

File: Loops/Loops Practice/index.html (hunk 1 above)
Comment (bfwetmore) [reply Loops/Loops Practice/index.html]: New commit pushed addressing this and directory comment above.

File: loops/Loops Practice/css/style.css [hunk 2]
```diff
@@ -41,4 +41,4 @@ h2 {
 p {
     font-size: 2em;
     line-height: 0.8;
-}
\ No newline at end of file
+}
```
Comment (do-work) [inline loops/Loops Practice/css/style.css]: For the directory we don't want to have spaces in it. So it should be
`loops/loopsPractice` or `loops/loops-practice` - I've seen examples of both.

**Review by do-work — CHANGES_REQUESTED** [review]

File: loops/Loops Practice/css/style.css (hunk 2 above)
Comment (bfwetmore) [reply loops/Loops Practice/css/style.css]: Fixed in latest commit

**Review by do-work — APPROVED** [review]

**Pull request merged**